adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/LoraConstants.java

```diff
@@ -55,6 +55,7 @@
     public static final String CODING_RATE = "coding_rate";
     public static final String FREQUENCY = "frequency";
     public static final String FRAME_COUNT = "frame_count";
+    public static final String APP_PROPERTY_MIC = "mic";
```

Comment:
can you please move this up to the other `APP_PROPERTY_`s

Reply:
done